data/names/ru.json

```diff
@@ -3447,31 +3535,50 @@
       "Винсон",
       "Винтер",
       "Витале",
-      "Виттен",
       "Витт",
+      "Виттен",
       "Вишневски",
+      "Вишневский",
+      "Вмятина",
```

Comment:
This is, hilariously, a translation of the name Dent as a common noun ("dent" as in a bump, e.g. on a personal vehicle). It would be transcribed Дэнт.